frontend/arthas-ai/app/teampage/page.tsx

```diff
@@ -0,0 +1,230 @@
+	},
+	{
+		id: "8",
+		image: "https://i.pinimg.com/originals/89/79/90/897990126794575981766f3c71324152.jpg",
+		name: "CHUU",
+		role: "Frontend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "9",
+		image: "https://i.pinimg.com/originals/89/79/90/897990126794575981766f3c71324152.jpg",
+		name: "CHUU",
+		role: "Frontend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "10",
+		image: "https://i.pinimg.com/originals/89/79/90/897990126794575981766f3c71324152.jpg",
+		name: "CHUU",
+		role: "Frontend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "11",
+		image: "https://i.pinimg.com/originals/89/79/90/897990126794575981766f3c71324152.jpg",
+		name: "CHUU",
+		role: "Frontend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "12",
+		image: "https://i.pinimg.com/originals/89/79/90/897990126794575981766f3c71324152.jpg",
+		name: "CHUU",
+		role: "Frontend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	}
+]
+
+{/* 5 Members */}
+const backend = [
+	{
+		id: "13",
+		image: "https://i.pinimg.com/originals/53/32/4b/53324bca1d5759d9c10c4b99847351ae.jpg",
+		name: "YVES",
+		role: "Backend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "14",
+		image: "https://i.pinimg.com/originals/53/32/4b/53324bca1d5759d9c10c4b99847351ae.jpg",
+		name: "YVES",
+		role: "Backend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "15",
+		image: "https://i.pinimg.com/originals/53/32/4b/53324bca1d5759d9c10c4b99847351ae.jpg",
+		name: "YVES",
+		role: "Backend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "16",
+		image: "https://i.pinimg.com/originals/53/32/4b/53324bca1d5759d9c10c4b99847351ae.jpg",
+		name: "YVES",
+		role: "Backend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	},
+	{
+		id: "17",
+		image: "https://i.pinimg.com/originals/53/32/4b/53324bca1d5759d9c10c4b99847351ae.jpg",
+		name: "YVES",
+		role: "Backend Developer",
+		quote: `"I worked on..."`,
+		github: "http://github.com/gabsebas",
+		linkedin: "https://linkedin.com/in/gabriela-sebastian-"
+	}
+]
+
+  const TeamPage = () => {
+	return (
```

Comment:
Code looks great! This comment is mostly for @quigongian. An option that we can explore is create a simple API route in the `actions.ts` file in the same directory and make an API request to the `actions.ts` file. Next.js will not ship `actions.ts` to client so we can do this. Plus it gives the benefit of giving @gabsebas @alishuhh @valentinaarojas experience with API calls and handling errors, loading states, etc.

Reply:
What is the api request returning?

Reply:
The API would simply return the teamLead arrays and the members array. But since it is in the Next.js app router, we don't have to do anything special but define an API route. We could discuss it more if you'd like.

I actually implemented a test route in this commit: https://github.com/arthasai/ArthasAI/pull/18/files#diff-22bcd583351170d9e38861adf842a2d02bc7dca32a48fce0db4324a8e20d11f3 look at the route.ts in the [doc]/API files (which is server side). 

Then you can hit the API endpoint by putting `fetch("/document/{doc-name}/API/")` in the client side.

If you want to use postman you make a GET to this URL: `http://localhost:3000/document/{doc-name}/API` it'll do the same thing.

Reply:
@alishuhh @valentinaarojas I want you to both, ONLY YOU BOTH, take a crack at this with pair programming.